virtwho/config.py

```diff
@@ -18,21 +18,30 @@
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """
 
+import collections
 import os
-
 from ConfigParser import SafeConfigParser, NoOptionError, Error, MissingSectionHeaderError
-from virtwho import DefaultInterval
+from virtwho import log
 from password import Password
 from binascii import unhexlify
 import hashlib
 import json
+import six
```

Comment:
Module `six` is not used. I would not mix this refactoring with effort to add support for Python 3.